azurelinuxagent/common/protocol/hostplugin.py

```diff
@@ -95,7 +95,7 @@ def __init__(self, endpoint):
         if not os.path.exists(self._get_fast_track_state_file()):
             self._supports_vm_settings = False
             self._supports_vm_settings_next_check = datetime.datetime.now()
-            self._fast_track_timestamp = None
+            self._fast_track_timestamp = timeutil.create_timestamp(datetime.datetime.min)
```

Comment:
does this cause opposite behavior where we want to skip the goal state if current goal state is outdated and fast_track_timestamp has min value. I don't know we ever get to that case, just throwing to think of any.
```
        if self._extensions_goal_state.created_on_timestamp < vm_settings_support_stopped_error.timestamp:
            self._extensions_goal_state.is_outdated = True
            msg = "Fetched a Fabric goal state older than the most recent FastTrack goal state; will skip it.\nFabric:    {0}\nFastTrack: {1}".format(
                  self._extensions_goal_state.created_on_timestamp, vm_settings_support_stopped_error.timestamp)
            self.logger.info(msg)
```